test/repo-sync.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 HELM_URL=https://storage.googleapis.com/kubernetes-helm
-HELM_TARBALL=helm-v2.6.2-linux-amd64.tar.gz
+HELM_TARBALL=helm-v2.9.1-linux-amd64.tar.gz
```

Comment:
🎉

Reply:
I can not, if you want.

Reply:
Noooooooo, 🎉 is a happy emoji. 2.9.1 is much better